pom.xml

```diff
@@ -149,24 +149,65 @@
             <version>1.3</version>
             <scope>test</scope>
         </dependency>
-
+	
+	<build>
+            <plugins>
+				<plugin>
+					<groupId>org.springframework.boot</groupId>
+					<artifactId>spring-boot-maven-plugin</artifactId>
+				</plugin>
+			</plugins>
+	</build>
```

Comment:
this won't work. The build section is in the dependencies section. 
Please test your code before asking for review